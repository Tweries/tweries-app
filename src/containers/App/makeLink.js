function makeLink(data) {
  return `https://twitter.com/${data.userId.split('|')[1]}/status/${
    data.statusIds[0]
  }`;
}

export default makeLink;
