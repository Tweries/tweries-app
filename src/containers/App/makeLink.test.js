import makeLink from './makeLink';

describe('makeLink', () => {
  const scenarios = [
    {
      statusIds: [
        '1230233654959407104',
        '1230233655626301442',
        '1230233656658157573'
      ],
      userId: 'twitter|1183836409850814464'
    }
  ];

  scenarios.forEach(data => {
    it('to match snapshot', () => {
      expect(makeLink(data)).toMatchSnapshot();
    });
  });
});
