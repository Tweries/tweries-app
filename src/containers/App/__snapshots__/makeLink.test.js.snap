// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`makeLink to match snapshot 1`] = `"https://twitter.com/1183836409850814464/status/1230233654959407104"`;
